Xamarin.Forms.Material.Android/MaterialPickerEditText.cs

```diff
@@ -33,6 +37,24 @@ protected override void OnFocusChanged(bool gainFocus, [GeneratedEnum] FocusSear
 			PickerManager.OnFocusChanged(gainFocus, this, (IPopupTrigger)Parent.Parent);
 		}
 
+		public override void SetText(ICharSequence text, BufferType type)
+		{
+			if (ViewCompat.IsLaidOut(this) && text != null)
+			{
+				int textWidth = Width - CompoundPaddingLeft - CompoundPaddingRight;
+
+				string fullText = text.ToString();
```

Comment:
The same is done in this renderer and in the PickerEditText, we could create an extension method or similar.

Reply:
Moved to an extension method in TextViewExtensions.